src/Components/FeelingRoom/fineAnimations.scss

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
Strange to see this show up here too, but shouldn't impact functionality or impact our marge.